packages/@sanity/components/src/dialogs/DefaultDialog.js

```diff
@@ -4,14 +4,15 @@ import React from 'react'
 import CloseIcon from 'part:@sanity/base/close-icon'
 import styles from 'part:@sanity/components/dialogs/default-style'
 import Button from 'part:@sanity/components/buttons/default'
+import ButtonCollection from 'part:@sanity/components/buttons/collection'
 import {Portal} from '../utilities/Portal'
 import Escapable from '../utilities/Escapable'
 import CaptureOutsideClicks from '../utilities/CaptureOutsideClicks'
 import Stacked from '../utilities/Stacked'
 
 export default class DefaultDialog extends React.Component {
   static propTypes = {
-    kind: PropTypes.oneOf(['default', 'warning', 'success', 'danger', 'info']),
+    color: PropTypes.oneOf(['default', 'warning', 'success', 'danger', 'info']),
```

Comment:
Looks like a breaking change, are you certain this doesn't break anything on our side?